src/js/components/artifacts/dialogs/artifactinformationform.js

```diff
@@ -106,7 +105,7 @@ export const ArtifactInformation = ({ customDeviceTypes, deviceTypes = [], name,
       <FormControl>
         <InputLabel htmlFor="release-name" style={{ alignItems: 'center', display: 'flex' }}>
           Release name
-          <Tooltip key="release-name-tip" title={<ReleaseTooltip />} placement="bottom" arrow={true} interactive leaveDelay={300}>
+          <Tooltip key="release-name-tip" title={<ReleaseTooltip />} placement="bottom" arrow leaveDelay={300}>
```

Comment:
you can imagine a component (such as `<Tooltip />` here) as a function, where the properties you pass to the component are the named arguments you pass in an object... so `<Tooltip arrow={true} />` becomes:
```js
function Tooltip(argumentsObject) {
  console.log(argumentsObject.arrow)
}
// or:
const Tooltip = ({ arrow }) => console.log(arrow)
```
by default any named argument that you pass is considered `true` - so the change in this line is mostly cosmetic... however... it might happen that a component doesn't follow that default and require an explicit boolean 😢... so when in doubt, be explicit...
If you don't pass anything, it would be like checking an undefined attribute:
```js
const foo = { };
console.log(foo.bar);
```
this will be `undefined`, so _falsey_ and thus in this particular case equal to `<Tooltip arrow={false} />` 👌

Does that clarify things?

Reply:
Yes :)

But you have still gone from `arrow=true` => `arrow=false`, and I don't see why? :thinking:

Reply:
> ... by default any named argument that you pass is considered `true` ...

so I went from `arrow={true}` (explicit) to `arrow={true}` (implicit)